### PR TITLE
Copyedit c9

### PR DESCRIPTION
**Pull request by jannes-m**
still working on the PR

### CODE REVIEW

File: 09-gis.Rmd [hunk 1]
```diff
@@ -23,34 +23,33 @@ CLIs are not unique to R.^[
...
 <!-- System commands can be sent from R with the `system()` and those of -->
 Many GISs originated as a CLI:
-it was only after the widespread uptake of computer mice and high resolution screens in the 1990s that GUIs became common.
+it was only after the widespread uptake of computer mice and high-resolution screens in the 1990s that GUIs became common.
 GRASS, one of the longest-standing GIS programs, for example, relied primarily on command-line interaction before it gained a sophisticated GUI [@landa_new_2008].
 ]
 In dedicated GIS packages, by contrast, the emphasis tends to be on the graphical user interface (GUI).
 You *can* interact with GRASS, QGIS, SAGA and gvSIG from system terminals and embedded CLIs such as the [Python Console in QGIS](https://docs.qgis.org/testing/en/docs/pyqgis_developer_cookbook/intro.html), but 'pointing and clicking' is the norm.
-This means many GIS users miss-out on the advantages of the command-line according to Gary Sherman, creator of QGIS [@sherman_desktop_2008]:
+This means many GIS users miss out on the advantages of the command-line according to Gary Sherman, creator of QGIS [@sherman_desktop_2008]:
```
Comment (Robinlovelace) [inline 09-gis.Rmd]: True, I should know better!

File: 09-gis.Rmd [hunk 2]
```diff
@@ -23,34 +23,33 @@ CLIs are not unique to R.^[
...
 <!-- Similar to GRASS, PostgreSQGL provides a (partial) GUI called [pgAdmin](https://www.pgadmin.org/) to facilitate the editing and administration of the database.  -->
 <!-- To make it clear, though PostGIS provides spatial functions, its main purpose is the handling of spatial objects in a relational database management system. -->
 <!-- Therefore, frequently users store their spatial data in PostGIS, and interact with it through a dedicated GIS software such as QGIS. Of course, you can also use R to access data from PostGIS (**sf**, **rgdal**, **rpostgis**). -->
-<!-- In summary,  a typical workflow would be: perform a large spatial query with PostGIS, then load the result into a further application (QGIS, R) for further geoprocessing.    -->
+<!-- In summary,  a typical workflow would be: perform a large spatial query with PostGIS, then load the result into a further application (QGIS, R) for further geoprocessing.-->
 ]
 The advantages of a good CLI such as that provided by R (and enhanced by IDEs such as RStudio) are numerous.
-A good CLI:
+A good CLI
```
Comment (Robinlovelace) [inline 09-gis.Rmd]: Nice style reducing the number of grammatical marks cluttering the page.

File: 09-gis.Rmd [hunk 3]
```diff
@@ -502,13 +501,13 @@ execGRASS("g.region", flags = c("quiet"),
           res = "1")
 ```
 
-Once you are familiar how to set up the GRASS environment, it becomes tedious to do so over and over again.
+Once you are familiar with how to set up the GRASS environment, it becomes tedious to do so over and over again.
```
Comment (Robinlovelace) [inline 09-gis.Rmd]: Alternative:

```
 Once you are familiar with setting up the GRASS environment, it becomes tedious to do so over and over again. 
```

Comment (Robinlovelace) [reply 09-gis.Rmd]: But happy with the change - both are right.

File: 09-gis.Rmd [hunk 4]
```diff
@@ -127,13 +126,13 @@ knitr::kable(x = d,
...
 Basically, functions `set_env()` and `open_app()` are doing this. 
 Note that it is optional to run `set_env()` and `open_app()` since all functions depending on their output will run them automatically if needed.
-Before running **RQGIS** make sure you have installed QGIS and all its (third-party) dependencies such as SAGA and GRASS.
+Before running **RQGIS**, make sure you have installed QGIS and all its (third-party) dependencies such as SAGA and GRASS.
 To install **RQGIS** a number of dependencies are required, as described in the [`install_guide`](https://cran.r-project.org/web/packages/RQGIS/vignettes/install_guide.html) vignette, which covers installation on Windows, Linux and Mac.
-At the time of writing (autumn 2018) RQGIS only supports the [Long Term Release](https://qgis.org/en/site/getinvolved/development/roadmap.html) (2.18) but support for QGIS 3 is in the pipeline.
+At the time of writing (autumn 2018) RQGIS only supports the [Long Term Release](https://qgis.org/en/site/getinvolved/development/roadmap.html) (2.18), but support for QGIS 3 is in the pipeline (see [RQGIS3](https://github.com/jannes-m/RQGIS3)).
```
Comment (Nowosad) [inline 09-gis.Rmd]: Coolcoolcool

Comment (Robinlovelace) [reply 09-gis.Rmd]: +1

**Pull request merged**